bin/runtime-common/src/messages.rs

```diff
@@ -205,24 +172,16 @@ pub mod source {
 		type Error = &'static str;
 
 		fn verify_message(
-			submitter: &OriginOf<ThisChain<B>>,
-			lane: &LaneId,
-			lane_outbound_data: &OutboundLaneData,
+			_submitter: &OriginOf<ThisChain<B>>,
+			_lane: &LaneId,
+			_lane_outbound_data: &OutboundLaneData,
 			_payload: &FromThisChainMessagePayload,
 		) -> Result<(), Self::Error> {
-			// reject message if lane is blocked
-			if !ThisChain::<B>::is_message_accepted(submitter, lane) {
-				return Err(MESSAGE_REJECTED_BY_OUTBOUND_LANE)
-			}
-
-			// reject message if there are too many pending messages at this lane
-			let max_pending_messages = ThisChain::<B>::maximal_pending_messages_at_outbound_lane();
-			let pending_messages = lane_outbound_data
-				.latest_generated_nonce
-				.saturating_sub(lane_outbound_data.latest_received_nonce);
-			if pending_messages > max_pending_messages {
-				return Err(TOO_MANY_PENDING_MESSAGES)
-			}
+			// IMPORTANT: any error that is returned here is fatal for the bridge, because
```

Comment:
I've removed max-pending-messages check here - it was set to `2^63` in RBH/WBH (iirc), so it doesn't work anyway.